openbb_platform/providers/fmp/openbb_fmp/models/cash_flow.py

```diff
@@ -24,80 +22,56 @@ class FMPCashFlowStatementQueryParams(CashFlowStatementQueryParams):
     Source: https://financialmodelingprep.com/developer/docs/#Cash-Flow-Statement
     """
 
-    period: Optional[Literal["annual", "quarter"]] = Field(
-        default="quarter",
-        description=QUERY_DESCRIPTIONS.get("period", ""),
-    )
-    cik: Optional[str] = Field(
-        default=None, description="Central Index Key (CIK) of the company."
-    )
-
-    @model_validator(mode="before")
-    @classmethod
-    def check_symbol_or_cik(cls, values):  # pylint: disable=no-self-argument
-        """Validate that either a symbol or CIK is provided."""
-        if values.get("symbol") is None and values.get("cik") is None:
-            raise ValueError("symbol or cik must be provided")
-        return values
+    period: Optional[Literal["annual", "quarter"]] = Field(default="annual")
 
 
 class FMPCashFlowStatementData(CashFlowStatementData):
     """FMP Cash Flow Statement Data."""
 
     __alias_dict__ = {
+        "period_ending": "date",
+        "fiscal_period": "period",
+        "fiscal_year": "calendarYear",
+        "filing_date": "fillingDate",
+        "accepted_date": "acceptedDate",
         "reported_currency": "reportedCurrency",
-        "net_cash_used_for_investing_activities": "netCashUsedForInvestingActivites",
-        "other_financing_activities": "otherFinancingActivites",
-        "net_cash_flow_from_operating_activities": "netCashProvidedByOperatingActivities",
-        "purchases_of_marketable_securities": "purchasesOfInvestments",
-        "sales_from_maturities_of_investments": "salesMaturitiesOfInvestments",
-        "payments_from_acquisitions": "acquisitionsNet",
-        "net_cash_flow_from_investing_activities": "netCashUsedForInvestingActivites",
-        "net_cash_flow_from_financing_activities": "netCashUsedProvidedByFinancingActivities",
+        "net_income": "netIncome",
+        "depreciation_and_amortization": "depreciationAndAmortization",
+        "deferred_income_tax": "deferredIncomeTax",
+        "stock_based_compensation": "stockBasedCompensation",
+        "change_in_working_capital": "changeInWorkingCapital",
+        "changes_in_account_receivables": "accountsReceivables",
+        "changes_in_inventory": "inventory",
+        "accounts_payables": "accountsPayables",
```

Comment:
this should be change_in_account_payable

Reply:
sorry, which one 😂

Reply:
@minhhoang1023 `accounts_payables` --> `change_in_account_payable` right?